### DISCUSSION TITLE
挂载腾讯云cos，今天突然用不了了

### DISCUSSION THREAD

**Discussion by xhhdd** [AlistGo/alist#1111] [Q&A]
alist版本号是v2.5.2-0-ge1ccc0b

用S3协议挂载的腾讯云cos，之前一直使用正常，今天突然报错。错误代码如下：

`RequestError: send request failed caused by: Get "https://aurality-1301707569.cos.ap-guangzhou.myqcloud.com/?delimiter=%2F&list-type=2&prefix=": read tcp 172.17.0.2:52050->159.75.57.69:443: read: connection reset by peer`

我使用了Custom Host，但是清空了Custom Host的内容之后还同样的报错内容。
已经确认过腾讯云cos那边可以正常访问

[这是我的网站链接](https://file.xhhdd.cc/)

谢谢大佬

**Comment by xhofe** [comment]
看报错信息似乎是网络问题，你可以尝试重启。

**Reply by xhhdd** [reply]
嗯嗯 我重启了服务器 也重新安装了alist的最新版。但是没法解决。
挂载的onedrive可以正常使用

我感觉可能是腾讯云的对象存储的问题，可能哪些设置改动了？
只要是腾讯云的都是这个问题 我换了好几个存储桶试了一下

不知道还有没有其他人有这个问题
我是突然出现的 
什么设置都没有改动

**Reply by xhhdd** [reply]
装了最新版之后解决了
